src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,15 +209,44 @@ function toggleCollapse(e, pans) {
     var expand = 'Expand';
     var collapse = 'Collapse';
     var panels = pans || $('div.panel-collapse');
+    var isElementAnExpandButton = $(e).html().trim().startsWith(expand);
+
+    if (isElementAnExpandButton) {
```

Comment:
Now I understand the code up to this line.  Perhaps the next line should be 
`expand(panels)`? That eliminates the need for the comment in line 215.